specification/eql/get/EqlGetRequest.ts

```diff
@@ -34,11 +36,13 @@ export interface Request extends RequestBase {
   }
   query_parameters: {
     /**
-     * Period for which the search and its results are stored on the cluster. Defaults to the keep_alive value set by the search’s EQL search API request.
+     * Period for which the search and its results are stored on the cluster.
+     * Defaults to the keep_alive value set by the search’s EQL search API request.
      */
     keep_alive?: Duration
     /**
-     * Timeout duration to wait for the request to finish. Defaults to no timeout, meaning the request waits for complete search results.
+     * Timeout duration to wait for the request to finish.
+     * @server_default no timeout
```

Comment:
Based on [an earlier review by @lcawl](https://github.com/elastic/elasticsearch-specification/pull/2171#discussion_r1255996084), I think we should only provide actual values as server defaults.

Reply:
Thanks for the info, I have removed that line via [1f85bcb](https://github.com/elastic/elasticsearch-specification/pull/2216/commits/1f85bcb7b8794640324e4af17e36588f3bd9cd7d).